### ISSUE TITLE
[Bug] cfn-lint throw E3002 for RotationPeriodInDays property in AWS::KMS::Key Resource

### ISSUE THREAD

**Issue by nalakawula** [aws-cloudformation/cfn-lint#3153]
### CloudFormation Lint Version

0.83.6

### What operating system are you using?

Ubuntu 22.04

### Describe the bug

cfn-lint throw E3002 for RotationPeriodInDays property in AWS::KMS::Key Resource
```sh
cfn-lint -I src/cloudformation/main.yml
2024-04-22 11:21:39,527 - cfnlint.runner - INFO - Run scan of template src/cloudformation/main.yml
E3002 Invalid Property Resources/KMSKey/Properties/RotationPeriodInDays
src/cloudformation/main.yml:34:7
```

Thank you

### Expected behavior

RotationPeriodInDays is valid property of AWS::KMS::Key Resource.

### Reproduction template

```yaml
Resources:
  KMSKey:
    Type: AWS::KMS::Key
    Properties:
      Description: KMS key
      EnableKeyRotation: true
      RotationPeriodInDays: 90
      KeyPolicy:
        Version: '2012-10-17'
        Id: my-test-kms-key
        Statement:
          - Sid: Allow direct access to key metadata to the root account
            Effect: Allow
            Principal:
              AWS: !Sub arn:aws:iam::${AWS::AccountId}:root
            Action: kms:*
            Resource: '*'
```

**Comment by kddejong** [comment]
Its in automated PR #3149 .  You can always run `cfn-lint --update-specs` until a new release is done (will be today).
